sig-network/concepts-and-designs/kubesphere-network-policy.md

```diff
@@ -48,23 +45,47 @@ spec:
         - namespaceSelector:
             matchLabels:
               "kubesphere.io/namespace": ${namespace-name}
-  Egress:
-    - To:
-        - namespaceSelector:
-            matchLabels:
-              "kubesphere.io/workspace": ${namespace-name}
+  policyTypes:
+  - ingress
+```
+
+Let's say I have two projects, A and B. They are in same workspace. If the A and B have not set annotation "kubesphere.io/network-isolate = enabled", they can access each other. The same thing goes for services. If project A set annotation "kubesphere.io/network-isolate = enabled", A cannot access services in B, unless A allows egress for services in B.
```

Comment:
If the A and B have not been set an annotation "kubesphere.io/network-isolate = enabled", they can access each other. The same thing goes for services. If project A has been set an annotation "kubesphere.io/network-isolate = enabled", A cannot access services in B, unless A allows egress for services in B.

Reply:
use the shadow for words like `"kubesphere.io/network-isolate = enabled"`